content/03.methods.md

```diff
@@ -14,9 +14,36 @@ talk about generative model and how it works
 ### Word Embeddings
 mention facebooks fasttext model and how we used it to train word vectors
 ### Discriminator Model
-talk about the discriminator model and how it works
+The discriminator model is designed to improve off of the generated labels from the generative model.
+We used a convolutional neural network with multiple filters as our discriminator model.
+This network uses multiple filters with fixed widths of 300 dimensions and can have varying heights (Figure {@fig:convolutional_network}).
```

Comment:
I don't quite understand this sentence. Is it about CNNs in general, or your specific one?

Reply:
This line is about my specific model.

Reply:
Here it says the heights can vary, but below it says 7. Should it just say a height of 7 `because this provided...`?

Reply:
Yeah I think saying height of 7 is appropriate here. Will change.